### PR TITLE
Machine broker, AWS driver and connection history

### PR DESCRIPTION
**Pull request by dynamiccast**

### CODE REVIEW

File: api/drivers/aws/driver.js [hunk 1]
```diff
@@ -0,0 +1,266 @@
...
+const pkgcloud = require('pkgcloud');
+const Promise = require('bluebird');
+const Driver = require('../driver');
+const ursa = require('ursa-purejs');
+const fs = Promise.promisifyAll(require('fs'));
+
+/**
+ * The port on which plaza will be listenning in the instances.
+ */
+const PLAZA_PORT = 9090;
```
Comment (Gentux) [inline api/drivers/aws/driver.js]: Plaza port is stored in environment variable for user storage. Shouldn't be the same here ?

Comment (dynamiccast) [inline api/drivers/aws/driver.js]: Good catch

File: api/models/Machine.js [hunk 2]
```diff
@@ -18,33 +18,94 @@
...
+    domain    : {
       type: 'string'
+    },
+    endDate   : {
+      type: 'datetime'
+    },
+    plazaport : {
+      type: 'integer'
+    },
+    user: {
```
Comment (Gentux) [inline api/models/Machine.js:67]: Padding before ':' should be the same as other field.

Comment (dynamiccast) [inline api/models/Machine.js:67]: Updated

File: README.md [hunk 3]
```diff
@@ -23,8 +23,17 @@ Some environment variable are expected to be set in `config.env`:
 - SMTP_SEND_FROM (defaults to mail@nanocloud.com) nanocloud's sender
 
 Manual driver specific:
-- EXECUTION_SERVERS (mandatory) the IP of the execution server
-- WINDOWS_PASSWORD (mandatory) the Windows password for the *administrator* account
+- MACHINES (array) Array of machine object to statically insert in the database
```
Comment (Gentux) [inline README.md:26]: How do we set machines' password with manual driver ?

File: api/hooks/broker.js [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+
+    /**
+     * Initialize the hook
+     *
+     * @method initialize
+     */
+    initialize(done) {
+      sails.after('hook:config:loaded', () => {
+        return MachineService.initialize()
+        .then(() => {
```
Comment (Gentux) [inline api/hooks/broker.js]: Shouldn't we indent ".then" block ?

File: README.md (hunk 3 above)
Comment (dynamiccast) [inline README.md:26]: Updated

**Pull request merged**